sdk/core/Azure.Core/samples/LongRunningOperations.md

```diff
@@ -2,7 +2,7 @@
 
 **NOTE:** Samples in this file apply only to packages that follow [Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html). Names of such packages usually start with `Azure`. 
 
-Some operations take long time to complete and require polling for their status. Methods starting long-running operations return `*Operation<T>` types.
+Some operations take long time to complete and require polling for their status. Methods starting long-running operations return either `*Operation<T>` or `PageableOperation<T>` types.
```

Comment:
Would they return a `PageableOperation<T>` directly or a subtype of it?

Reply:
Good point.  I will update to `*PageableOperation<T>`